recipes/rtg-tools/meta.yaml

```diff
@@ -1,13 +1,13 @@
 package:
   name: rtg-tools
-  version: "3.6"
+  version: "3.7.1"
 
 source:
-  fn: rtg-tools-3.6-nojre.zip
-  url: https://github.com/RealTimeGenomics/rtg-tools/releases/download/3.6/rtg-tools-3.6-nojre.zip
+  fn: rtg-tools-3.7.1-nojre.zip
+  url: https://github.com/RealTimeGenomics/rtg-tools/releases/download/3.7.1/rtg-tools-3.7.1-nojre.zip
 
 build:
-  number: 1
+  number: 0
   skip: False
 
 requirements:
```

Comment:
java-jdk is deprecated, change the dependency to `openjdk`.
also check if the package requires a specific jvm version.